app/blueprints/facebook_post_blueprint.rb

```diff
@@ -13,23 +13,36 @@ class FacebookPostBlueprint < Blueprinter::Base
   association :user, blueprint: FacebookUserBlueprint
 
   field :image_file do |post|
-    unless post.image_file.nil?
+    if post.image_file.nil? == false && post.aws_image_keys.blank?
       file = File.open(post.image_file).read
       Base64.encode64(file)
     end
   end
 
   field :video_file do |post|
-    unless post.video_file.nil?
+    if post.video_file.nil? == false && post.aws_video_key.blank?
       file = File.open(post.video_file).read
       Base64.encode64(file)
     end
   end
 
   field :video_preview_image_file do |post|
-    unless post.video_preview_image_file.nil?
+    if post.video_preview_image_file.nil? == false && post.aws_video_preview_key.blank?
       file = File.open(post.video_preview_image_file).read
       Base64.encode64(file)
     end
   end
+
+  field :aws_video_key do |post|
+    post.aws_video_key
```

Comment:
In JavaScript, the difference between `methodName` and `methodName()` is whether you are referencing the function or invoking it. Does the same apply w/Ruby?

I ask because it looks like we're inconsistent throughout all the Blueprints with these key fields. Here and on `aws_image_keys`, we don't invoke. On `aws_video_preview_key`, we do. Then in the Instagram post blueprint, we always invoke. On the YouTube post blueprint, we again do both.

If that's intentional, great! But worth sanity-checking.